week_1/Array/move_zeroes/move_zeroes.test.js

```diff
@@ -0,0 +1,7 @@
+const moveZeroes = require("./move_zeroes");
+
+test("move_zeroes moves zeroes to the end of an array of numbers while maintaining the relative order of the remaining elements", () => {
+	let nums = [0, 10, 0, 2, 3, 4, 0, 5, 6, 7];
```

Comment:
can we add a test that asserts nothing is changed when there are no zeros, or when the zeros are already on the end?

Reply:
Done